packages/components/src/components/Tooltip/index.tsx

```diff
@@ -5,22 +5,67 @@ import Tippy, { TippyProps } from '@tippyjs/react/headless';
 import { Instance, Props as TProps } from 'tippy.js';
 import { transparentize } from 'polished';
 import { Link } from '../typography/Link';
+import { Icon } from '../Icon';
+import { Fade } from 'react-awesome-reveal';
```

Comment:
`react-awesome-reveal` is dependency of `suite-web-landing` only. Not sure if it's useful to introduce it in components package.

I feel like we have too many animation libraries used :-D ( react-spring, framer-motion, css animations, react-awesome-reveal, ...)

Reply:
67dcf05

Reply:
how is this addressed in the fixup? the usage of react-awesome-reveal was just moved from packages/components to packages/suite, right? So it should be defined as dependency in `packages/suite/package.json`?

Reply:
yes the usage was moved. Shall I `yarn add react-awesome-reveal` also in packages/suite?

Reply:
I think so.

Reply:
added in 1a9e24e28b84e41ea5232603340c00fc50714bbe